Bump op_version_set

zdevito opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#19812 Bump op_version_set**

A previous PR changed the way that fork was serialized, so we need
to bump op_version_set so that older versions know they cannot load it.

Also add the missing version number check and a test for it.

Differential Revision: [D15102803](https://our.internmc.facebook.com/intern/diff/D15102803)

zdevito committed:
Bump op_version_set

A previous PR changed the way that fork was serialized, so we need
to bump op_version_set so that older versions know they cannot load it.

Also add the missing version number check and a test for it.

zdevito committed:
Update on "Bump op_version_set"

Bump op_version_set

A previous PR changed the way that fork was serialized, so we need
to bump op_version_set so that older versions know they cannot load it.

Also add the missing version number check and a test for it.

gh-metadata: pytorch pytorch 19812 gh/zdevito/33/head

apaszke commented on torch/csrc/jit/import_source.cpp:
```diff
@@ -133,22 +121,39 @@ struct SourceImporter {
+  // Our tests generate a bunch of code where the version number is not
+  // set. In this case, we just choose the most up-to-date version.
+  // Actual archives should always set the version.
+  return CURRENT_OP_VERSION_SET;
```
This looks like a bad idea to me. We should fix the test to not do dumb things, not make our code do them.

zdevito committed:
Update on "Bump op_version_set"

Bump op_version_set

A previous PR changed the way that fork was serialized, so we need
to bump op_version_set so that older versions know they cannot load it.

Also add the missing version number check and a test for it.

gh-metadata: pytorch pytorch 19812 gh/zdevito/33/head
